tests/unit/test_synthDrivers/test_espeak.py

```diff
@@ -0,0 +1,83 @@
+# A part of NonVisual Desktop Access (NVDA)
+# This file is covered by the GNU General Public License.
+# See the file COPYING for more details.
+# Copyright (C) 2022 NV Access Limited.
+
+"""Unit tests for the eSpeak synth driver submodule.
+"""
+
+import logging
+import unittest
+import logHandler
+from speech.commands import LangChangeCommand
+from source.synthDrivers.espeak import SynthDriver
+
+
+class FakeESpeakSynthDriver:
+	_language = "default"
+	_defaultLangToLocale = {"default": "en-gb"}
+	availableLanguages = {"fr", "fr-fr", "en-gb", "ta-ta"}
+
+
+class TestSynthDriver(unittest.TestCase):

```

Comment:
Perhaps a docstring to explain, this is a test of the logic in our internal espeak driver, rather than an integration test with the espeak-ng binaries.

Reply:
This testing class contains both integration and internal testing. I will split these up and add appropriate docstrings